src/NuGet.Indexing/GalleryServiceImpl.cs

```diff
@@ -82,13 +39,13 @@ public static string QuerySearch(NuGetSearcherManager searcherManager, string q,
             }
         }
 
-        private static string DocumentCountImpl(IndexSearcher searcher, Query query, Filter filter)
+        static void DocumentCountImpl(JsonWriter jsonWriter, IndexSearcher searcher, Query query, Filter filter)
```

Comment:
John asked me to keep consistent style in these existing pieces of code. I do not have a strong preference but I can see the point of keeping individual files self consistent.